meta-dts-distro/recipes-dts/dts/dts/dts

```diff
@@ -114,7 +128,9 @@ while : ; do
   check_os_version
   echo
   echo "  1) Dasharo HCL report - dump hardware information from this device"
-  echo "  2) Install Dasharo firmware"
+  if ! look_for_dasharo; then
```

Comment:
@esesyn01 this would need to be changed after we change the function name

Reply:
@TomaszAIR Done - https://github.com/Dasharo/meta-dts/pull/25/commits/22b1c2c1ec48f722fcfd8901d10439bc1b9ecf4b#diff-d2bd602c4c837aa03dd0dd7eb140ddf24397c7c33b1961b4b83d04f014010d64R116